assets/js/common.js

```diff
@@ -894,6 +894,8 @@ var hivepress = {
 								if (!messageContainer.is(':empty')) {
 									messageContainer.addClass(messageClass + '--error').show();
 								}
+							}else if (response.hasOwnProperty('redirect')){
+								window.location.replace(response.redirect);
```

Comment:
This one is not needed, details below. Please always let me know if you change or extend the API (e.g. add new undocumented parameters, helpers, hooks).